src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -569,161 +559,133 @@
+          "$ref": "#/definitions/tool"
+        },
+
+        "invocation": {
+          "description": "A string containing the runtime parameters with which the tool was invoked. For command line tools, this string may consist of the completely specified command line used to invoke the tool.",
           "type": "string"
         },
 
-        "files": 
-        {
-          "description": "A dictionary each of whose keys is a URI and each of whose values is an array of fileReference objects representing the location of a single file target scanned during the run.",
+        "files": {
+          "description": "A dictionary each of whose keys is a URI and each of whose values is an array of file objects representing the location of a single file target scanned during the run.",
           "type": "object",
           "additionalProperties": {
             "type": "array",
             "items": {
-              "$ref": "#/definitions/fileReference"
+              "$ref": "#/definitions/fileData"
             }
           }
         },
 
-        "runStartTime": 
-        {
-          "description": "An optional string specifying the date and time at which the run started.",
+        "results": {
+          "description": "The set of results contained in an SARIF log. The results array can be omitted when a run is merely exporting rules metadata. It must be present (but may be empty) in the event that a log file represents an actual scan.",
+          "type": "array",
+          "minItems": 0,
+          "items": {
+            "$ref": "#/definitions/result"
+          }
+        },
+
+        "rules": {
+          "description": "An array of rule descriptor objects that describe all rules associated with an analysis tool or a specific run of an analysis tool.",
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/rule"
+          }
+        },
+
+        "startTime": {
+          "description": "A string specifying the date and time at which the run started.",
           "type": "string",
           "format": "date-time"
         },
 
-        "runEndTime": 
-        {
-          "description": "An optional string specifying the date and time at which the run ended.",
+        "endTime": {
+          "description": "A string specifying the date and time at which the run ended.",
           "type": "string",
           "format": "date-time"
         },
 
-        "correlationId": 
-        {
-          "description": "An optional string identifier that allows the run to be associated with a larger automation process.",
+        "correlationId": {
+          "description": "An identifier that allows the run to be associated with a larger automation process.",
```

Comment:
I don't like "correlated within". Things are "correlated with" other things. How about:

"An identifier that allows the run to be correlated with other artifacts within a larger automation process."